src/applications/check-in/components/WhatToDoNext.jsx

```diff
@@ -92,15 +93,15 @@ const WhatToDoNext = props => {
                               })}
                             </li>
                           );
-                        })
-                      ) : (
-                        <li data-testid="appointment-bullet">
-                          {t('day-of-week-month-day-time', {
-                            date: new Date(appointment.startTime),
-                          })}
-                        </li>
-                      )}
-                    </ul>
+                        })}
+                      </ul>
+                    ) : (
+                      <div data-testid="appointment-bullet">
```

Comment:
I think you might have to change this to a paragraph tag so that it gets the correct font size. There was a design system change that adjusts font size on sans serif fonts but only if they are wrapped in certain tags. This came up in this [issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90154).

Reply:
updated and updated screenshots